Release/src/http/common/http_helpers.cpp

```diff
@@ -88,7 +88,7 @@ size_t chunked_encoding::add_chunked_delimiters(_Out_writes_(buffer_size) uint8_
 #ifdef _WIN32
         sprintf_s(buffer, sizeof(buffer), "%8IX", bytes_read);
 #else
-        snprintf(buffer, sizeof(buffer), "%8zX", bytes_read);
+        snprintf(buffer, sizeof(buffer), "%8X", (std::uint32_t)bytes_read);
```

Comment:
Why not use `static_cast` instead of the old c-style cast?

Reply:
To me it looks like C-code (maybe because I'm a C guy) - but I have no strong opinion about it

Reply:
@BrandonStaab ping

Reply:
I think a `static_cast` is more appropriate here since the old c-style cast has the potential to do much more, and potentially unwanted things.

Reply:
Done